src/event.js

```diff
@@ -677,30 +683,24 @@ jQuery.Event.prototype = {
 	isImmediatePropagationStopped: returnFalse,
 
 	preventDefault: function() {
-		var e = this.originalEvent;
-
 		this.isDefaultPrevented = returnTrue;
 
-		if ( e && e.preventDefault ) {
-			e.preventDefault();
+		if ( this.originalEvent ) {
```

Comment:
True but it would only break if someone returned `false` from their custom event handler or explicitly called `preventDefault` or `stopPropagation` on that custom event. Given the amount of code out there I could see it happening though.